app/src/Lib/FileSystem/Filesystem.php

```diff
@@ -8,8 +8,10 @@ interface Filesystem
 {
     public const DOIL_PATH_MAIN_CONFIG = "/etc/doil";
     public const DOIL_PATH_LIB = "/usr/local/lib/doil";
-    public const DOIL_PATH_SHARE = "/usr/local/share/doil";
     public const DOIL_PATH_BIN = "/usr/local/bin/doil";
+    public const DOIL_PATH_SHARE = "/usr/local/share/doil";
+    public const DOIL_PATH_SHARE_STACK = "/usr/local/share/doil/stack";
```

Comment:
The two added constants are not used anywhere, right?

Reply:
Done!